.gitlab-ci.yml

```diff
@@ -125,6 +125,14 @@ before_script:
   variables: &ci-template-vars
     TEST_TARGET: "$CI_JOB_NAME"
 
+.ci-template-flambda: &ci-template-flambda
+  <<: *ci-template
+  dependencies:
+    - build:be+flambda
+  variables:
+    OPAM_SWITCH: "be"
```

Comment:
You need to `<<: *ci-template-vars`.
All the data is invalid except for color: TEST_TARGET doesn't get set so we run `make -f Makefile.ci` and the default target is color.

Reply:
(comment is shown outdated because it's made on a single commit and the diff with all commits looks different)

Reply:
Ahh, working so late can never be good. Let's try again-

Reply:
> You need to <<: *ci-template-vars.

My lack of knowledge of yaml bite here; I assumed that `<<: *ci-template` would make the latter variable declaration "additive". Let's try again.